build_all.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -e 
+set -x
+
+script_dir=$(dirname "$0")
+projects=(
+qemu-lite
```

Comment:
Nit: would be nice if this list was sorted ;)

Reply:
I did it in that order because qemu and the kernel takes longer to build in `OBS`, so give it a bit more time to  try the finish in a similar time. Added comments now.